website/docs/guides/markdown-features/markdown-features-inline-toc.mdx

```diff
@@ -5,15 +5,17 @@ description: Using inline table-of-contents inside Docusaurus Markdown
 slug: /markdown-features/inline-toc
 ---
 
-TODO(erickzhao): add maxHeadingLevel docs import BrowserWindow from '@site/src/components/BrowserWindow';
+import BrowserWindow from '@site/src/components/BrowserWindow';
 
 Each Markdown document displays a tab of content on the top-right corner.
 
 But it is also possible to display an inline table of contents directly inside a markdown document, thanks to MDX.
 
 ## Full table of contents {#full-table-of-contents}
 
-The `toc` variable is available in any MDX document, and contain all the top level headings of a MDX document.
+The `toc` variable is available in any MDX document, and contains all the headings of a MDX document.
+
+By default, only h2 and h3 variables will be shown in the TOC, but you can change which heading levels will be show by setting `maxHeadingLevel` or `minHeadingLevel`.
```

Comment:
```suggestion
By default, only `h2` and `h3` headings are displayed in the TOC.
You can change which heading levels are visible by setting `maxHeadingLevel` or `minHeadingLevel`.
```

How about this?

- Use monospaced font for `h2` and `h3`.
- Rewrite sentences to be easier to understand and read.